modules/Components/src/client/ACASFormFields.coffee

```diff
@@ -462,8 +468,9 @@ class window.ACASFormLSThingInteractionFieldController extends ACASFormAbstractF
 		@clearError()
 		if @userInputEvent
 			thingID = @thingSelectController.getSelectedID()
+			thingCodeName = @thingSelectController.getSelectedCode()
 			if thingID?
-				@getModel().setItxThing id: thingID
+				@getModel().setItxThing id: thingID, codeName: thingCodeName
```

Comment:
This got mixed into this pull request but was really mean as a feature add for ThingComboSelect boxes to be able to get the thing code along with the id.